recipes/opus/all/opus_buildtype.cmake

```diff
@@ -0,0 +1,23 @@
+# Set a default build type if none was specified
```

Comment:
Why do you need this if Conan by default sets Release?

Reply:
I have no idea, bincrafters' recipe legacy again.

Reply:
this whole file is working around an upstream bu: https://github.com/xiph/opus/issues/129